package.json

```diff
@@ -3,16 +3,24 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^2.0.1",
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.6.2",
+    "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-redux": "^8.1.2",
+    "react-router-dom": "^6.15.0",
     "react-scripts": "5.0.1",
+    "redux-logger": "^3.0.6",
+    "redux-mock-store": "^1.5.4",
+    "uuid": "^9.0.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=5173 react-scripts start",
```

Comment:
- [ ] I'd advice you don't specify a static port for which your applicatio should be run. Considering the fact anybody can clone your repo, recruiter included, it will be great to avoid clashes of ports in use.

- [ ] Currently, i'm not able to start your app due to this change alone. Please un-bind the port so it removes this bug and make sure potential CORES issues if fixed so your backend and frontend can communicate.

- Good job 👏 

### View:
![image](https://github.com/mohammadnewton/hello-react-front-end/assets/50990925/0fcc1ada-bd56-480b-9983-08dae98e1f42)